db/migrate/20160228140111_update_params_priority.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateParamsPriority < ActiveRecord::Migration
+  def change
```

Comment:
It's probably better to have a `def up` with this and `def down` that puts back the previous priorities to make sure everything goes back to the correct priority.